src/app/App.js

```diff
@@ -16,6 +17,7 @@ const Video = React.lazy(() => import('../pages/Video/Video'))
 export function App() {
   return (
     <React.Suspense fallback="Loading">
+      <Navbar />
```

Comment:
create a `Layout` component that contains `Navbar` (and other layout elements such as `Footer`) then wrap `<React.Suspense />` into it